lib/dal/src/builtins/schema/test_exclusive_schema_fallout.rs

```diff
@@ -0,0 +1,211 @@
+use si_pkg::{
+    ActionFuncSpec, AttrFuncInputSpec, AttrFuncInputSpecKind, FuncArgumentSpec, FuncSpec,
+    FuncSpecBackendKind, FuncSpecBackendResponseType, PkgSpec, PropSpec, SchemaSpec,
+    SchemaVariantSpec, SchemaVariantSpecData, SiPkg, SocketSpec, SocketSpecData, SocketSpecKind,
+};
+use si_pkg::{FuncSpecData, SchemaSpecData};
+
+use crate::func::argument::FuncArgumentKind;
+use crate::func::intrinsics::IntrinsicFunc;
+use crate::pkg::import_pkg_from_pkg;
+use crate::{prop::PropPath, ActionKind, PropKind};
+use crate::{BuiltinsResult, DalContext};
+
+pub async fn migrate_test_exclusive_schema_fallout(ctx: &DalContext) -> BuiltinsResult<()> {
```

Comment:
Same exact file, but this sole function is no longer a method off of the `MigrationDriver`